src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -7,6 +7,25 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        if (value == 0) {
+            return "0";
+        }
+        StringBuilder result = new StringBuilder();
+        int power = 0;
+        while (Math.pow(2, power) <= value) {
```

Comment:
please read the first topic "Converting decimal integer to binary" - https://indepth.dev/the-simple-math-behind-decimal-binary-conversion-algorithms/
try simplifying solution using this algorithm